src/page/ListPage/TodoPage.jsx

```diff
@@ -7,9 +7,14 @@ export const TodoPage = () => {
   const [todos, setTodos] = useState(defaultTodos);
 
   useEffect(() => {
-    const storedTodos = JSON.parse(localStorage.getItem("todos"));
-    if (storedTodos) {
-      setTodos(storedTodos);
+    try {
+      const storedTodos = JSON.parse(localStorage.getItem("todos"));
+      if (storedTodos) {
+        setTodos(storedTodos);
+      }
+    } catch (e) {
+      console.error(e);
+      setTodos(defaultTodos);
     }
```

Comment:
so , I recommedn this `try-catch` structure here. this line

```js
JSON.parse()
```

might raise an error and we should be resiliant 💪